hogql_parser/string.cpp

```diff
@@ -56,3 +60,18 @@ string unquote_string_terminal(antlr4::tree::TerminalNode* node) {
     throw ParsingError(e.what(), node->getSymbol()->getStartIndex(), node->getSymbol()->getStopIndex() + 1);
   }
 }
+
+string unquote_string_chunk_terminal(antlr4::tree::TerminalNode* node, bool escape_quotes) {
```

Comment:
The functions in here could use comments equivalent to the Python docstrings, getting harder to grasp them